apps/metagpt/cli.py

```diff
@@ -0,0 +1,198 @@
+import os
+from typing import Any
+
+import click
+import openai
+import structlog
+
+from etl.paths import BASE_DIR
+
+# Initialize logger
+log = structlog.get_logger()
+
+# Example of new metadata format
+NEW_METADATA_EXAMPLE = (
+    BASE_DIR / "snapshots" / "emissions" / "2023-11-23" / "national_contributions_annual_emissions.csv.dvc"
+)
+
+# Additional instructions or configurations
+ADDITIONAL_INSTRUCTIONS = """
+Metadata Field Guidelines:
+
+1. attribution (string, optional)
+   - Capital letter start, except names like 'van Haasteren'.
+   - No ending period.
+   - Ends with year of date_published in parenthesis.
+   - No semicolons.
+
+2. attribution_short (string, recommended)
+   - Use if producer name is long; shorten the producer name in an informative way.
+   - Capital letter start, except names like 'van Haasteren'.
+   - No ending period.
+   - Refers to producer or well-known data product, not year.
+   - Use acronym if well-known.
+
+3. citation_full (string, required)
+   - Capital letter start.
+   - Ends with period.
+   - Includes year of publication.
+   - Match producer's format with minor edits.
+   - List multiple sources for compilations.
+
+4. date_accessed (string, required)
+   - Format: YYYY-MM-DD.
+   - Reflects access date of current version.
+
+5. date_published (string, required)
+   - Format: YYYY-MM-DD or YYYY.
+   - Reflects publication date of current version.
+
+6. description (string, recommended)
+   - Capital letter start and period end.
+   - Avoid other metadata fields unless crucial.
+   - Succinct description of data product.
+
+7. description_snapshot (string, recommended)
+   - Capital letter start and period end.
+   - Define if data product and snapshot differ.
+   - Describe snapshot specifically.
+
+8. license (string, required)
+   - Standard license names or producer's custom text.
+   - CC BY 4.0 if unspecified, pending confirmation.
+
+9. license.url (string, required if existing)
+   - Complete URL to license on producer's site.
+   - Avoid generic license pages.
+
+10. producer (string, required)
+    - Capital letter start, except names like 'van Haasteren'.
+    - No ending period, except 'et al.'.
+    - Exclude dates, semicolons, OWID references.
+
+11. title (string, required)
+    - Capital letter start.
+    - No ending period.
+    - Identify data product, not snapshot.
+
+12. title_snapshot (string, required if different)
+    - Capital letter start.
+    - No ending period.
+    - Use if snapshot differs from data product.
+
+13. url_download (string, required if existing)
+    - Direct download URL or S3 URI.
+
+14. url_main (string, required)
+    - URL to data product's main site.
+
+15. version_producer (string or number, recommended if existing)
+    - Use producer's version naming.
+"""
+
+
+# Main CLI command setup with Click
+@click.command()
+@click.option("--path-to-file", prompt=True, type=str, help="Path to the metadata file.")
```

Comment:
It'd be nice to have the option for overwriting input file, i.e.
```
@click.option(
    "--overwrite",
    default=False,
    type=bool,
    help="Overwrite input file if set to True. Otherwise create a new file with prefix `updated_with_gpt_`.",
)
```